sicp-doc/sicp-pict.scrbl

```diff
@@ -275,21 +360,29 @@ Returns a painter that rotates the image.}
 @defproc[(beside [p1 painter/c] [p2 painter/c]) painter/c]{
 Constructs a painter that paints the images side-by-side.}
 
-@defproc[(below [p1 painter/c] [p2 painter/c]) painter/c]{
-Constructs a painter that paints the first image
-below the second.}
+@defproc[(beside3 [p1 painter/c] [p2 painter/c] [p3 painter/c]) painter/c]{
+Constructs a painter that paints the images one beside the other.}
+
+@defproc[(above [p1 painter/c] [p2 painter/c]) painter/c]{
+Constructs a painter that paints the first image above the second.}
 
 @defproc[(above3 [p1 painter/c] [p2 painter/c] [p3 painter/c]) painter/c]{
 Constructs a painter that paints the images one above the other.}
 
-@defproc[(superpose [p1 painter/c] [p2 painter/c]) painter/c]{
-Constructs a painter that paints the two images
-on top of each other.}
+@defproc[(below [p1 painter/c] [p2 painter/c]) painter/c]{
+Constructs a painter that paints the first image below the second.}
+
+@defproc[(superpose [p painter/c] ...) painter/c]{
+Constructs a painter that paints all images on top of each other.}
 
```

Comment:
Ah! Here it is.

Don't feel obliged, but it would be great to have simple examples for `below` and friends.
It's much easier to understand from a visual example than from the prose.